packages/react-query/src/internals/context.tsx

```diff
@@ -133,60 +134,50 @@ export interface TRPCContextState<
    * @link https://react-query.tanstack.com/guides/query-invalidation
    */
   invalidateQueries: (
-    pathAndInput?: [string, unknown?] | string,
+    queryKey: QueryKey,
     filters?: InvalidateQueryFilters,
     options?: InvalidateOptions,
   ) => Promise<void>;
 
   /**
    * @link https://react-query.tanstack.com/reference/QueryClient#queryclientresetqueries
    */
-  resetQueries:
-    | ((
-        pathAndInput?: [string, unknown?] | string,
-        filters?: ResetQueryFilters,
-        options?: ResetOptions,
-      ) => Promise<void>)
-    | ((filters?: ResetQueryFilters, options?: ResetOptions) => Promise<void>);
```

Comment:
overload here doesn't work (I think??) since we always infer the 1st arg as the queryKey